client-src/rundump.c

```diff
@@ -160,14 +162,17 @@ main(
 
```

Comment:
You are missing `{` here.
Since the block below has multiple statements, the `else` above matches only the first one
and `validate_dump_option(argc, argv);` is always executed. This breaks legitimate `xfsdump` usage, because it aborts when using the `-F` option.